concordium-consensus/src/Concordium/Scheduler/DummyData.hs

```diff
@@ -114,7 +114,18 @@ cdi11keys = readSigningKeys . BSL.fromStrict $ $(makeRelativeToProject "testdata
 -- {-# WARNING cdi13 "Do not use in production." #-}
 -- cdi13 :: CredentialDeploymentInformation
 -- cdi13 = readCredential . BSL.fromStrict $ $(makeRelativeToProject "testdata/credential-12.json" >>= embedFile)
-
+{-# WARNING cdi14 "Do not use in production." #-}
+-- same registration id as cdi3, but invalid IP
+cdi14 :: AccountCreation
+cdi14 = readAccountCreation . BSL.fromStrict $ $(makeRelativeToProject "testdata/credential-14.json" >>= embedFile)
+{-# WARNING cdi15 "Do not use in production." #-}
+-- same registration id as cdi3, but invalid AR
+cdi15 :: AccountCreation
+cdi15 = readAccountCreation . BSL.fromStrict $ $(makeRelativeToProject "testdata/credential-15.json" >>= embedFile)
+{-# WARNING cdi16 "Do not use in production." #-}
+-- same registration id as cdi3, but invalid key size
+cdi16 :: AccountCreation
+cdi16 = readAccountCreation . BSL.fromStrict $ $(makeRelativeToProject "testdata/credential-16.json" >>= embedFile)
```

Comment:
Please update documention on how these credentials are meant to be generated.

Here https://github.com/Concordium/concordium-node/tree/main/concordium-consensus/testdata#test-data-generation

Reply:
Ideally you'd update the generate-testdata tool.

Reply:
These were actually not used anymore so I deleted them.